src/styles/info-cell.less

```diff
@@ -1,21 +1,21 @@
-.DocumentInfoCell {
+.InfoCell {
   display: block;
   text-decoration: none;
 }
 
-.DocumentInfoCell-title {
+.InfoCell-mainText {
   color: @edu-content-title-color;
   font-size: 1.25em;
   font-weight: bold;
 }
 
-.DocumentInfoCell-description {
+.InfoCell-description {
   margin-top: 0.2em;
   color: fade(@edu-text-color, 80%);
 }
 
-.DocumentInfoCell-dates {
+.InfoCell-subtext {
   margin-top: 0.75em;
   font-size: 0.9em;
   color: @edu-text-color-secondary;
-}
+}
```

Comment:
You seem to have removed the final new line character (also in the other LESS file), maybe you can check if your editor picks up the `.editorconfig` file correctly.

Reply:
Must have been a strange glitch, I just opened and re-saved the files now and the line was added. Updated both!